lib/computeStatistics.js

```diff
@@ -0,0 +1,182 @@
+'use strict';
+var Cesium = require('cesium');
+var isDataUri = require('./isDataUri');
+var PrimitiveHelpers = require('./PrimitiveHelpers');
+
+var defined = Cesium.defined;
+
+module.exports = {
+    getStatistics : getStatistics,
+    printStatistics : printStatistics
+};
+
+/**
+ * Creates an object type for glTF Statistics.
+ *
+ * @property {Number} buffersSizeInBytes The total size of all the buffers.
+ * @property {Number} numberOfImages The number of images part of the assest.
+ * @property {Number} numberOfExternalRequests The number of external requests to fetch the assest data.
+ * @property {Number} numberOfDrawCalls The number of draw calls required in the asset.
+ * @property {Number} numberOfRenderedPrimitives The number of rendered primitives.
```

Comment:
For consistency: `The total number of rendered primitives in the asset`. Also add a note that this typically means the number of triangles.